pset-functions.js

```diff
@@ -31,6 +31,22 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+function scores(playerCardScore, dealerCardScore){
+   if (playerCardScore > 21 && dealerCardScore > 21){
+    return "0"
+  } else if (playerCardScore <= 21 || dealerCardScore > 21){
```

Comment:
Clo, be a little careful with the || operator; so long as one side is true, the whole equation is true. In a situation where the playerCardScore is 19 and the dealerCardScore is 20, the dealer should win but the current logic returns 19 (playScore). This is a quick fix, just switch all the || with && and your code works flawlessly. Really great use of if/else blocks to catch the different situations in proper sequence.